drivers/sensor/bmm150/Kconfig

```diff
@@ -0,0 +1,96 @@
+# Kconfig - BMM150 Geomagnetic sensor configuration options
+
+#
+# Copyright (c) 2016 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+menuconfig BMM150
+	bool "BMM150 I2C Geomagnetic Chip"
+	depends on SENSOR && I2C
+	default n
+	help
+	  Enable driver for BMM150 I2C-based Geomagnetic sensor.
+
+config BMM150_DEV_NAME
+	string "BMM150 device name"
+	depends on BMM150
+	default "bmm150"
+
+config BMM150_I2C_ADDR
+	hex "BMM150 I2C slave address"
+	default 0x10
+	depends on BMM150
+	help
+	  Specify the I2C slave address for the BMM150
+
+config BMM150_I2C_MASTER_DEV_NAME
+	string "I2C master where BMM150_MAGN is connected"
+	depends on BMM150
+	default "I2C_0"
```

Comment:
@MaureenHelm  Can you please reply and review further driver code so that I can merge for 1.9 ?